dependencies/required_extra.txt

```diff
@@ -7,7 +7,7 @@ smac4nni
 
 # BOHB
 ConfigSpace>=0.4.17
-statsmodels==0.12.0
+statsmodels>=0.12.0
```

Comment:
@liuzhe-lz  Glad to see the requirement on statsmodels is no longer pinned.  This was causing issues for us.  Just want to know if there are plans for a pypi release in the near future that would include this change.  That would be very helpful

Reply:
link #4454